tests/lib/preflights/sanity_checks.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for the Sanity Checker preflight."""
+
+import unittest
+
+from dftimewolf.lib import state
+from dftimewolf.lib.errors import DFTimewolfError
+from dftimewolf.lib.preflights import sanity_checks
+from dftimewolf import config
+
+
+class SanityChecks(unittest.TestCase):
+  """Tests for the Sanity Checker preflight."""
+
+  def testInitialization(self):
+    """Test initialisation of the sanity checker."""
+    test_state = state.DFTimewolfState(config.Config)
+    checker = sanity_checks.SanityChecks(test_state)
+    self.assertIsNotNone(checker)
+
+  def testInvalidDateOrder(self):
+    """Test that dates in the incorrect order produces an error."""
+    test_state = state.DFTimewolfState(config.Config)
+    checker = sanity_checks.SanityChecks(test_state)
+
+    start_date = '2020-10-31'
+    end_date = '2020-10-01'
+    expected_error = sanity_checks.DATE_ORDER_ERROR_STRING.format(
+        start_date, end_date)
```

Comment:
During tests, it's always better to use as literal values as possible, to limit the amount of logic in the test (even if it's simple) but also to make the test very clear to whoever's reading it. (just like you did in `testInvalidDateFormat`)

I would just call `SetUp` with hardcoded params: `SetUp(startdate='2020-10-31', end_date='2020-10-01', dateformat=...)` and compare the resulting exception with the appropriate string literal.

Reply:
Done.